docs/porting/psa/spm.md

```diff
@@ -0,0 +1,260 @@
+<h2 id="spm-port">PSA SPM</h2>
+
+Secure Partition Manager (SPM) is a part of the PSA Firmware Framework that is responsible for isolating software in partitions, managing the execution of software within partitions and providing interprocessor communication (IPC) between partitions.
+
+For more information about SPM, please refer to [the SPM overview page](/docs/development/apis/mbed-psa.html).
+
+**This page gives guidelines for silicon partners wishing to have Secure Partition Manager capabilities**
+
+### New target configuration
+
+When adding a new target, a new root target node should be added to mbed-os/targets/targets.json file.
+For PSA support, specific PSA related fields should be defined for this target:
+
+1. Secure target must inherit from `SPE_Target` meta-target.
+2. Nonsecure target must inherit from `NSPE_Target`.

```

Comment:
@AnotherButler  what is the right way of writing, `nonsecure` or `non-secure`?
If it is a single word we probably need to change our configuration symbols.
In PSA FF specs the dash version (`non-secure`) is used.